common/src/main/java/org/broadleafcommerce/common/util/RuntimeLog4jConfigurer.java

```diff
@@ -36,12 +36,12 @@ public String getLog4jConfigLocation() {
         return log4jConfigLocation;
     }
 
-    public void setLog4jConfigLocation(String log4jConfigLocation) {
-        this.log4jConfigLocation = log4jConfigLocation;
-        try {
-            Log4jConfigurer.initLogging(log4jConfigLocation);
-        } catch (FileNotFoundException e) {
-            throw new RuntimeException(e);
-        }
-    }
+//    public void setLog4jConfigLocation(String log4jConfigLocation) {
```

Comment:
need to think do we need this class at all ?
maybe check later versions of framework or looks like it was deprecated in favor of
[org.springframework.util.Log4jConfigurer](https://docs.spring.io/spring-framework/docs/4.2.x/javadoc-api/org/springframework/util/Log4jConfigurer.html)
as of Spring 4.2.1, in favor of Apache Log4j 2 (following Apache's EOL declaration for log4j 1.x)